### DISCUSSION TITLE
Disable Auto-pause/play

### DISCUSSION THREAD

**Discussion by MrClaritas** [timschneeb/GalaxyBudsClient#221] [General]
Hello,
First of all, I wanted to thank you for the galaxy buds client, really appreciated it!

I wondered if it's possible for an option that disables sensors so it doesn't stop playing music/video or something else whenever I take my buds off.

**Comment by timschneeb** [comment]
Unfortunately, that's not possible. They don't have any (hidden) setting for this.

Some operating systems like Linux allow you to intercept and suppress Bluetooth media control events, but this doesn't seem to be the case with Windows.
